Lib/defcon/objects/contour.py

```diff
@@ -224,6 +226,16 @@ def __len__(self):
         return len(self._points)
 
     def __getitem__(self, index):
+        if isinstance(index, slice):
+            if index.start > len(self._points):
+                raise IndexError
+            if index.stop is None:
+                stop = len(self._points)
+            elif index.stop > len(self._points):
+                raise IndexError
+            else:
+                stop = index.stop
+            return self._points[index.start:stop]
```

Comment:
I think this checking for `IndexError` when slicing is both wrong and unnecessary. 

https://docs.python.org/3.5/tutorial/introduction.html

> out of range slice indexes are handled gracefully when used for slicing

We should simply pass on the the `start`, `stop`, as well as the  `step` attribute (which is currently missing here), to the self._points list, which knows how to handle slices.